code/game/objects/random/medicine.dm

```diff
@@ -3,20 +3,55 @@
 	icon_state = "meds-green"
 
 /obj/random/medical/item_to_spawn()
-	return pickweight(list(/obj/item/stack/medical/bruise_pack = 4,\
-				/obj/item/stack/medical/ointment = 4,\
-				/obj/item/stack/medical/advanced/bruise_pack = 2,\
-				/obj/item/stack/medical/advanced/ointment = 2,\
-				/obj/item/stack/medical/splint = 1,\
-				/obj/item/bodybag = 2,\
-				/obj/item/bodybag/cryobag = 1,\
-				/obj/item/weapon/storage/pill_bottle/kelotane = 2,\
-				/obj/item/weapon/storage/pill_bottle/antitox = 2,\
-				/obj/item/weapon/storage/pill_bottle/tramadol = 2,\
+	return pickweight(list(/obj/item/stack/medical/bruise_pack = 5,\
```

Comment:
insert /obj/random/pills/maint into this list with a high weight (15-20) since it represents a pool of content

Also it seems like you've added bottles of liquid chemicals here. Can you apply the same random treatment to those?